types-requirements.txt

```diff
@@ -1,5 +1,5 @@
 mypy
-mypy-protobuf
+mypy-protobuf<v3.4
```

Comment:
Issue states that mypy-protobuf `3.4.0` is valid, should we change the requirement to be inclusive `3.4`?

Reply:
Could I ask if there is a reason you are prefixing with `v`? The release on pypi states without `v`, but it seems `pip` accepts this..

Reply:
Good comment, will change it.